app/defines/random.h

```diff
@@ -23,7 +23,6 @@ unsigned int get_random_int(){
 // generate pseudo random number between 0 - 127
 unsigned int get_random_0_127(){
 
-    //return get_random_int() & 0x7F;
```

Comment:
This may be the better option. ```return get_random_int() & 0x7F;```
I think the GCC should be able to optimize "% 128" to have similar performance.. 

What ever you want is fine

Reply:
I leave it that way, 0x7F is also 127, if we want numer between 0-127 I think we have to use 0x80